### DISCUSSION TITLE
Глок-17 в массы!

### DISCUSSION THREAD

**Discussion by octothorpteam** [octothorp-team/public-dobrograd#590] [Предложения]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/lillshycringe/32/182_2.png) [lillshycringe](https://forum.octothorp.team/u/lillshycringe) ([#4660](https://forum.octothorp.team/p/4660))

У обычных офицеров полиции появился глок-17, что делает их гораздо сильнее в бою, хотя у них итак есть определенные баффы, а именно: возможность забиндить достование из холстера, ускоренное доставание. Предлагаю заменить старый глок на 17 или создать новое оружие у токаря, так-как с новой системой стрельбы глок стал просто мусором.

- Я за!
- Я против!

0голосов

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/jusko/32/2001_2.png) [jusko](https://forum.octothorp.team/u/jusko) ([#4664](https://forum.octothorp.team/p/4664))


Сейчас пистолеты слабые в целом, поэтому глок и является самым худшим и неюзабельным оружием на сервере. Против его добавления в массы, для гражданских есть аналог в виде п228. Раньше, если я не ошибаюсь, как раз таки был на сервере глок 17, но его заменили на 27-ой по каким-то причинам.

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/warsucksletsparty/32/2109_2.png) [warsucksletsparty](https://forum.octothorp.team/u/warsucksletsparty) ([#4754](https://forum.octothorp.team/p/4754))


![](https://forum.octothorp.team/user_avatar/forum.octothorp.team/lillshycringe/40/182_2.png) lillshycringe:


> появился глок-17, что делает их гораздо сильнее в бою

![](https://forum.octothorp.team/user_avatar/forum.octothorp.team/lillshycringe/40/182_2.png) lillshycringe:


> так-как с новой системой стрельбы глок стал просто мусором.

что…?

**Reply by travka4000** [reply]
что блин?

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/lillshycringe/32/182_2.png) [lillshycringe](https://forum.octothorp.team/u/lillshycringe) ([#4821](https://forum.octothorp.team/p/4821))


Глок-17 и обычный глок это разные оружия. Во второй цитате я говорил про глок с девятью патронами.

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/freedomchechnya/32/1265_2.png) [FreedomChechnya](https://forum.octothorp.team/u/FreedomChechnya) ([#5030](https://forum.octothorp.team/p/5030))


Против. Мы играем РП а не ПВП, так что баланс не к чему, все как в реале. Полицейские в ИРЛ лучше обучены и имеют лучшее снаряжение и экипировку, также и тут на сервере. Если мы будем балансить крайм и полицию, то пострадают в первую очередь цивилы, которым легче самим начать отстреливаться, чем ждать пока приедут копы, ничем не превосходящие краймеров, разве что в численности.

PS.
 Как по-моему задача краймеров не перестрелять копов, а не допустить перестрелки с ними, остаться не замеченными, после чего успешно скрыться с места преступления. Наказанием за провал ограбления и служит приезд полиции, которая гораздо сильнее вас.

**Comment by octothorpteam** [comment]
## ![](https://forum.octothorp.team//user_avatar/forum.octothorp.team/kustik/32/467_2.png) [Kustik](https://forum.octothorp.team/u/Kustik) ([#5049](https://forum.octothorp.team/p/5049))


Против по одной причине: Glock 27 модели является, грубо говоря, “Палкой, которая стреляет раз в год”. Он дешёвый, даёт хоть какой-то шанс на убийство/самооборону.

* * *

Да, он плох в нанесении увечий, но если добавить токарю возможность крафта Glock 17, то всё просто забьют на другие пистолеты.

**Comment by LENFYS** [comment]
За.
